### PR TITLE
feat: Settled view.

### PR DESCRIPTION
**Pull request by Tulun**
Settled view.
Screen Shot 2022-01-28 at 1 46 19 PM

2) Adjusted time remaining.
https://app.shortcut.com/uma-project/story/4293/once-a-proposal-happens-the-liveness-period-reads-2hr-total-xyz-remaining-lets-update-to-time-remaining-xyz-remove-unnecessar

3) Login to connect wallet.
https://app.shortcut.com/uma-project/story/4298/update-verbiage-from-login-to-connect-wallet

4) Update low balance to insufficient funds.
https://app.shortcut.com/uma-project/story/4282/update-verbiage-from-low-balance-to-insufficient-funds

### CODE REVIEW

File: src/components/request/Request.tsx [hunk 1]
```diff
@@ -2,10 +2,18 @@ import { useEffect } from "react";
...
+  Wrapper,
+  TableContentWrapper,
+  TableSection,
+  TableTitle,
+} from "./Request.styled";
 import RequestHero from "./RequestHero";
 import useClient from "hooks/useOracleClient";
-
+import useReader from "hooks/useOracleReader";
+import { oracle } from "@uma/sdk";
```
Comment (daywiss) [inline src/components/request/Request.tsx:14]: I dont think u ever need to import this, you can get it from the useoracleclient hook

File: src/components/request/SettledTable.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      <Body>
+        {rows.map((row, ridx) => {
+          return (
+            <Row key={ridx}>
+              {row.cells.map((cell, cidx) => {
+                return (
+                  <StyledCellBody
+                    className={cell.cellClassName ?? ""}
+                    key={cidx}
+                    size={cell.size}
```
Comment (daywiss) [inline src/components/request/SettledTable.tsx:62]: Can the settled table return an id with the row rather than using this index? Or maybe use the value

**Review by daywiss — COMMENTED** [review]
Looks great, nice to see reuse of table. Just some minor nits

File: src/components/request/SettledTable.tsx (hunk 2 above)
Comment (Tulun) [reply src/components/request/SettledTable.tsx:62]: For what purpose sorry?

File: src/components/request/Request.tsx (hunk 1 above)
Comment (Tulun) [reply src/components/request/Request.tsx]: Changed!

**Review by daywiss — APPROVED** [review]

**Pull request merged**